scipy/optimize/optimize.py

```diff
@@ -1998,6 +1998,9 @@ def brent(func, args=(), brack=None, tol=1.48e-8, full_output=0, maxiter=500):
     Uses inverse parabolic interpolation when possible to speed up
     convergence of golden section method.
 
+    brent does not ensure that the located minimum is within the initially
```

Comment:
- [ ] Improve Short Description
- [ ] Improve notes description
- [ ] Add Examples

Anything else I need to do?

> So perhaps the difference is that fminbound is okay with returning an endpoint of the interval, whereas brent really wants to find a local minimum.

I'll check this too